### ISSUE TITLE
HorizonFrame no longer existing in ctapipe

### ISSUE THREAD

**Issue by moralejo** [cta-observatory/cta-lstchain#85]
https://github.com/cta-observatory/cta-lstchain/blob/25e3caf889c3bde8e031021a1877d2cb7257a0d9/lstchain/reco/utils.py#L15

This fails  with:  ImportError: cannot import name 'HorizonFrame'
I think HorizonFrame is no longer existing in ctapipe.

**Comment by rlopezcoto** [comment]
related to #80

**Comment by thomasgas** [comment]
> [cta-lstchain/lstchain/reco/utils.py](https://github.com/cta-observatory/cta-lstchain/blob/25e3caf889c3bde8e031021a1877d2cb7257a0d9/lstchain/reco/utils.py#L15)
> 
> Line 15 in [25e3caf](/cta-observatory/cta-lstchain/commit/25e3caf889c3bde8e031021a1877d2cb7257a0d9)
>  from ctapipe.coordinates import HorizonFrame 
> 
> This fails with: ImportError: cannot import name 'HorizonFrame'
> I think HorizonFrame is no longer existing in ctapipe.

Hi @moralejo! this is because in ctapipe we decided to drop the HorizonFrame, since it's the same thing as the AltAz in astropy. So you can replace the:
```python
from ctapipe.coordinates import HorizonFrame
```
with
```python
from astropy.coordinates import AltAz
```

**Comment by rlopezcoto** [comment]
Solved in #87
